beta-src/src/state/game/game-api-slice.ts

```diff
@@ -469,6 +523,10 @@ const gameApiSlice = createSlice({
             setCommand(state, command, "mapCommands", "all");
           }
         }
+      } else if (inProgress && method === "dblClick") {
+        if (subsequentClicks.length) {
+          console.log("support move");
```

Comment:
I saw you say that you'd remove the other console log on the next pr, just wanted to note that there is one here as well

Reply:
Yea I'll remove them on the next PRs as these console logs are helping me find my way as I'm working through support mvoe.